app/src/main/java/com/eventyay/organizer/core/faq/list/FaqListPresenter.java

```diff
@@ -33,56 +28,43 @@
     private final List<Faq> faqs = new ArrayList<>();
     private Faq previousFaq = new Faq();
     private final FaqRepository faqRepository;
-    private final DatabaseChangeListener<Faq> faqChangeListener;
     private final Map<Faq, ObservableBoolean> selectedMap = new ConcurrentHashMap<>();
     private boolean isContextualModeActive;
 
     @Inject
-    public FaqListPresenter(FaqRepository faqRepository, DatabaseChangeListener<Faq> faqChangeListener) {
+    public FaqListPresenter(FaqRepository faqRepository) {
         this.faqRepository = faqRepository;
-        this.faqChangeListener = faqChangeListener;
     }
 
     @Override
     public void start() {
         loadFaqs(false);
-        listenChanges();
     }
 
     @Override
     public void detach() {
         super.detach();
         selectedMap.clear();
-        faqChangeListener.stopListening();
     }
 
     public void loadFaqs(boolean forceReload) {
         getFaqSource(forceReload)
             .compose(dispose(getDisposable()))
             .compose(progressiveErroneousRefresh(getView(), forceReload))
+            .flatMapIterable(faqs -> faqs)
             .toList()
```

Comment:
Remove last 2 lines

Reply:
Will have to create a new function in ViewTransformers then, with `ObservableTransformer` instead of `SingleTransformer`. Is that fine?